nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -155,7 +157,20 @@ def _get_axis(
         if not target_point.is_weight_target_point():
             return 1
         node = nncf_graph.get_node_by_name(target_point.target_node_name)
-        return node.metatype.weight_definitions.weight_channel_axis
+
+        weight_channel_axis = node.metatype.weight_channel_axis
+        if node.layer_attributes.has_node_attrs():
+            if node.metatype == ONNXGemmMetatype:
```

Comment:
Why should the `MinMaxAlgoBackend` calculate this? Why not calculate it while building `NNCFGraph`?

Reply:
Because NNCFGraph is used not only for MinMax

Reply:
I don't see any problem to calculate it for `NNCFGraph` while building. For example, the `SmoothQuant` algorithm also uses information about the `weight_channel_axis`.

Reply:
This is the same as it is done in OpenVINO. I aligned with this logic and don't see any problem with that.
If you thnk that it should be redone you concern only about ONNX?